src/main/java/com/stripe/model/OrderItem.java

```diff
@@ -1,11 +1,13 @@
 package com.stripe.model;
 
-public class OrderItem extends StripeObject {
+import com.stripe.net.APIResource;
```

Comment:
So since order items _are_ API resources, I think it's perfectly okay to change their subclass here.

We probably should make `ExpandableField` available on `StripeObject` though.